README.md

```diff
@@ -250,14 +250,14 @@ GitHub workflow.
   
 **GitHub Rate Limit:** As per observation, due to rate limit restriction of GitHub REST API calls, we may observe fewer pull requests to be created.
 
-
 ## Additional Parameters
-|Input Parameter |Description                              |
-|-----------------|------------------------------------------|
-|`synopsys_bridge_path`| Provide a path, where you want to configure or already configured Synopsys Bridge. [Note - If you don't provide any path, then by default configuration path will be considered as - $HOME/synopsys-bridge]. If the configured Synopsys Bridge is not the latest one, latest Synopsys Bridge version will be downloaded          |
-| `bridge_download_url`      | Provide URL to bridge zip file. <br> If provided, Synopsys Bridge will be automatically downloaded and configured.               |
-|`bridge_download_version`| Provide bridge version.<br> If provided, the specified version of Synopsys Bridge will be automatically downloaded and configured.             |
-| `include_diagnostics`      | Synopsys Bridge diagnostics files will be available to download when it is set to `true`. Additionally `diagnostics_retention_days` can be passed as integer value between 1 to 90 to retain the files (Be default file be available for 90 days).               |
+| Input Parameter                     | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|-------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `synopsys_bridge_install_directory` | Provide a path, where you want to configure or already configured Synopsys Bridge. [Note - If you don't provide any path, then by default configuration path will be considered as - $HOME/synopsys-bridge]. If the configured Synopsys Bridge is not the latest one, latest Synopsys Bridge version will be downloaded                                                                                                                                           |
```

Comment:
If the configured Synopsys Bridge is not the latest one, latest Synopsys Bridge version will be downloaded---> remove this line.